sample.env

```diff
@@ -92,7 +79,7 @@ CSM_EMAIL_APP_USERNAME=
 CSM_EMAIL_APP_PASSWORD=
 CSM_EMAILS_TO_SEND_TO=
 CSM_TARGET_SMTP_SERVER_ADDRESS=
-CSM_TARGET_SMTP_SERVER_PORT=587
+CSM_TARGET_SMTP_SERVER_PORT=

```

Comment:
I am not against making this blank, but it might be useful to have a comment that the common ports for SMTP are 587, 25, 465 and 2525

Reply:
This was a change that I made in the other PR, this is reverting that change back to what the develop branch has